loco_translate.module

```diff
@@ -16,3 +19,64 @@ function loco_translate_theme($existing, $type, $theme, $path) {
     ],
   ];
 }
+
+/**
+ * Implements hook_cron().
+ *
+ * Automation of Loco Translate module.
+ */
+function loco_translate_cron() {
+  $now = time();
+  $config = \Drupal::config('loco_translate.settings');
+  $utility = \Drupal::service('loco_translate.utility');
+
+  // Get configured langcode to pull from loco.
+  $langcodes = $config->get('automation.pull.langcodes');
+  if ($langcodes) {
+    foreach ($langcodes as $langcode => $status) {
+      // Skip unchecked langcode.
+      if (!$status) {
+        continue;
+      }
+
+      // Check last run of this langcode.
+      $last_run = $utility->getLastPull($langcode);
```

Comment:
we need to be sure that it's an integer. From the DockBlock `getLastPull` return a string.

Reply:
casted into int